src/ucm/cuda/cudamem.c

```diff
@@ -225,6 +236,11 @@ static ucm_cuda_func_t ucm_cuda_driver_funcs[] = {
 #if CUDA_VERSION >= 11020
     UCM_CUDA_FUNC_ENTRY(cuMemFreeAsync),
 #endif
+    /**
+     * cudaGetSymbolAddress is in ucm_cuda_driver_funcs list because
+     * hook for cuModuleGetGlobal doesn't intercept cudaGetSymbolAddress call.
+     * */
+    UCM_CUDA_FUNC_ENTRY(cudaGetSymbolAddress),
```

Comment:
why not place it in runtime_api_hooks? we install both driver and runtime hooks anyway..

Reply:
maybe make some changes in the hooks logic or array names then? it seems weird that a runtime function is in the "driver" API list